org.Bhupesh-V.ugit.yml

```diff
@@ -0,0 +1,45 @@
+app-id: org.Bhupesh-V.ugit
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: ugit
+finish-args:
+  - --filesystem=home:rw
+  - --share=network
+modules:
+  - name: fzf
+    buildsystem: simple
+    build-commands: 
+      - tar -xvf fzf-0.33.0-linux_amd64.tar.gz
```

Comment:
1. this is not how you handle tarballs
2. you are unpacking x86_64 binary, yet a aarch64 will still be built. 
3. you should build this from source.